### ISSUE TITLE
[Bug] Couldn’t find object

### ISSUE THREAD

**Issue by FlashHit** [Powback/MapEditor#322]
```

[2022-01-30 04:33:30+01:00] [info] [VeniceEXT] [mapeditor] [GameObjectManager] ERROR: Object with id 939C85D5-0B7D-B56B-A6A0-7B45FCD06FFA does not exist 
[2022-01-30 04:33:30+01:00] [info] [VeniceEXT] [mapeditor] [ServerTransactionManager] WARNING: Failed to execute command: SetTransformCommand 
[2022-01-30 04:33:30+01:00] [info] [VeniceEXT] [mapeditor] [GameObjectManager] ERROR: Object with id 2A7119D3-96BC-83A4-0042-FDB3BAF4DF15 does not exist 
[2022-01-30 04:33:30+01:00] [info] [VeniceEXT] [mapeditor] [ServerTransactionManager] WARNING: Failed to execute command: SetTransformCommand 
[2022-01-30 04:33:30+01:00] [info] [VeniceEXT] [mapeditor] [GameObjectManager] ERROR: Object with id EC27644F-99E5-96D9-801D-505181480ADD does not exist 
[2022-01-30 04:33:30+01:00] [info] [VeniceEXT] [mapeditor] [ServerTransactionManager] WARNING: Failed to execute command: SetTransformCommand 
[2022-01-30 04:33:30+01:00] [info] [VeniceEXT] [mapeditor] [GameObjectManager] ERROR: Object with id 524DA42B-9BA5
```
Might be only the refactor branch. Not sure.

**Comment by 3ti65** [comment]
this was fixed no? @FoolHen

**Comment by FoolHen** [comment]
Ah yeah, fixed in ca9b14b435d99f1ed146376c39a7300a961a8278
